test/inputs/priority_input_test.rb

```diff
@@ -5,14 +5,14 @@ class PriorityInputTest < ActionView::TestCase
   test 'input generates a country select field' do
     with_input_for @user, :country, :country
     assert_select 'select#user_country'
-    assert_select 'select option[value=Brazil]', 'Brazil'
+    assert_select 'select option[value=BR]', 'Brazil'
     assert_no_select 'select option[value=""][disabled=disabled]'
   end
 
   test 'input generates a country select with SimpleForm default' do
     swap SimpleForm, country_priority: [ 'Brazil' ] do
       with_input_for @user, :country, :country
-      assert_select 'select option[value=""][disabled=disabled]'
+      assert_select 'select option[value="---------------"][disabled=disabled]'
```

Comment:
I think it doesn't, but sounds like the `priority_divider` default from `country_select` changed between 1.x and 2.x.